[Bug] 手动curl请求是ok的，但是使用ais提示ERR99999，无法测评性能

### 操作系统及版本

openeuler22.03sp4

### 安装工具的python环境

docker容器中的python环境

### python版本

3.11

### AISBench工具版本

3.1.20260330

### AISBench执行命令

ais_bench --models vllm_api_stream_chat --datasets synthetic_gen -m perf

### 模型配置文件或自定义配置文件内容

/usr/local/lib/python3.11/site-packages/ais_bench/benchmark/configs/models/vllm_api/vllm_api_stream_chat.py
/usr/local/lib/python3.11/site-packages/ais_bench/benchmark/configs/datasets/synthetic/synthetic_gen.py

### 预期行为

_No response_

### 实际行为

[2026-04-14 00:42:59,231] [ais_bench] [^[[32mINFO^[[0m] Start warmup, run with concurrency: 1
^MWarmup:   0%|          | 0/1 [00:00<?, ?case/s]^MWarmup: 100%|██████████| 1/1 [00:00<00:00, 2526.69case/s]
Traceback (most recent call last):
  File "/usr/local/lib/python3.11/site-packages/ais_bench/benchmark/tasks/openicl_api_infer.py", line 704, in <module>
    raise e
  File "/usr/local/lib/python3.11/site-packages/ais_bench/benchmark/tasks/openicl_api_infer.py", line 701, in <module>
    inferencer.run(task_state_manager)
  File "/usr/local/lib/python3.11/site-packages/ais_bench/benchmark/tasks/openicl_api_infer.py", line 488, in run
    self.warm_up(data_list, task_state_manager)
  File "/usr/local/lib/python3.11/site-packages/ais_bench/benchmark/tasks/openicl_api_infer.py", line 467, in warm_up
    raise AISBenchRuntimeError(
ais_bench.benchmark.utils.logging.exceptions.AISBenchRuntimeError: [^[[1m^[[41m^[[33mTINFER-RUNTIME-001^[[0m]^[[1m^[[31mwarmup failed^[[0m. Exit task because all warmup requests failed, failed reasons: {'': 1} | Visit ^[[4m^[[35mhttps://ais-bench-benchmark.readthedocs.io/zh-cn/latest/faqs/error_codes.html#tinfer-runtime-001^[[0m for further help.
[2026-04-14 00:42:59,251] [ais_bench] [^[[33mWARNING^[[0m] Task state is error, exit loop
[ERROR] 2026-04-14-00:42:59 (PID:25698, Device:-1, RankID:-1) ERR99999 UNKNOWN application exception

### 前置检查

- [x] 我已读懂主页文档的快速入门，无法解决问题
- [x] 我已检索过FAQ，无重复问题
- [x] 我已搜索过现有Issue，无重复问题
- [x] 我已更新到最新版本，问题仍存在

--- a/ais_bench/benchmark/openicl/icl_inferencer/icl_base_api_inferencer.py
+++ b/ais_bench/benchmark/openicl/icl_inferencer/icl_base_api_inferencer.py
@@ -165,6 +165,7 @@ class BaseApiInferencer(BaseInferencer):
             connector=aiohttp.TCPConnector(limit=concurrency + 1),
             timeout=aiohttp.ClientTimeout(total=get_request_time_out()),
             max_line_size=get_max_chunk_size(),
+            trust_env=True,
         )
         warmup_tasks = []
         async def warmup_limited_request_func(data: dict):
@@ -411,7 +412,8 @@ class BaseApiInferencer(BaseInferencer):
         connector = aiohttp.TCPConnector(limit=num_workers + 1)
         timeout = aiohttp.ClientTimeout(total=get_request_time_out())
         session = aiohttp.ClientSession(
-            connector=connector, timeout=timeout, max_line_size=get_max_chunk_size()
+            connector=connector, timeout=timeout, max_line_size=get_max_chunk_size(),
+            trust_env=True,
         )
         self.logger.debug(f"Create aiohttp session with "
                      f"connector: {connector}, "

